### DISCUSSION TITLE
"Lost connection to host or Steam network"

### DISCUSSION THREAD

**Discussion by cherpak1** [SmartlyDressedGames/Unturned-3.x-Community#2315] [Q&A]
tried to run my own server with steamcmd. everything is seemed to be fine, ports are opened, server hosted but the issue is i can't join to my server. ping is stable, like 20-40. what i have to do?

**Comment by SDGNelson** [comment]
Did you open ports 27015, 27016 and 27017?
Perhaps you have already read it, but if not I would recommend looking at: https://github.com/SmartlyDressedGames/U3-Docs/blob/master/ServerHosting.md

**Reply by cherpak1** [reply]
yep. 27015-27032. 27015, 27016, 27017 opened, tried all three. starting from 27018-27032 the server is not joinable, only lan.

**Reply by cherpak1** [reply]
5 years ago i've been running own server without any problems, full online with 24 players. u know, copy unturned folder to the desktop, bat, plugins, etc. but now with cmd something went wrong. the other players can join and play without any problems.

**Reply by SDGNelson** [reply]
There was a similar case where someone was logging into steamcmd using a steam account, rather than login anonymous, and then forgetting to log back into Steam. Are you using login anonymous (recommended)?

**Reply by cherpak1** [reply]
yep. "login anonymous", "app_update 1110390" command too. then just a safe "quit" command

**Reply by SDGNelson** [reply]
Hmm... I am not sure what else to suggest then, sorry. There have been a few similar cases while trying to get hosts to use the dedicated server app instead of the client files, and each time it turned out to be a weird aspect of their setup (e.g. not using login anonymous).

For what it's worth it should be fully supported - i.e. I can host a local dedicated server on my PC, connect to it, and have other players connect via the internet, but there are always some weird differences with each host's setup.

**Reply by cherpak1** [reply]
one dude suggested me to connect with "localhost" thru the "connect" section. I'll try this evening.

**Reply by cherpak1** [reply]
yep, you shoul've connect using "localhost" "<port>" and it's working well. just pin that, if someone'll have the same issue.

**Reply by SDGNelson** [reply]
Okay, glad to hear you got it working.
